piqe_ocp_lib/api/resources/ocp_prometheus_client.py

```diff
@@ -0,0 +1,115 @@
+import requests
```

Comment:
This is a client and not a native Kubernetes or Openshift resource, so I don't know that this should live under api/resources

Reply:
Right but currently we don't have any other dir structure so i kept it here. May be can we create ocp_monitoring dir inside lib and move this code in there? May be health check as well?